components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -660,6 +660,19 @@ def leave_none_unset_int(s):
           " ] or [\"Repository\", {\"viewname\": \"webindex\", "
           "\"query_string\": {\"experimenter\": -1}}, "
           "{\"title\": \"Repo\"}]'``")],
+    "omero.web.ui.metadata_pane":
```

Comment:
something simmilar could be implemented in Insight, should these settings go to server as `omero.client.ui.metadata_pane`